packages/core/src/lib/markdown-it/plugins/markdown-it-block-embed/services/PowerPointOnlineService.js

```diff
@@ -5,15 +5,15 @@ const VideoServiceBase = require("./VideoServiceBase");
 class PowerPointOnlineService extends VideoServiceBase {
 
   getDefaultOptions() {
-    return {width: 610, height: 481};
+    return { width: 500, height: 375 };
   }
 
   extractVideoID(reference) {
     return reference;
   }
 
   getVideoUrl(serviceUrl) {
-    return `${serviceUrl}&action=embedview&wdAr=1.3333333333333333`;
+    return `${serviceUrl}`;
```

Comment:
Hmm what's the removal for? I would think the `wdAr` parameter is important for the powerpoint service to render things correctly, if the default isn't already `1.333...`. If it's not necessary, let's remove the change.

Reply:
Actually from my testing it seems as if the wdAr does not actually do anything to the aspect ratio of the PowerPoint, perhaps its due to the way things are implemented (but im not entirely sure why it doesn't affect anything). 

That's why I made the decision to remove it, since keeping it around did nothing anyways.

Reply:
Got it, while it can be tempting to clean these sort of things up, let's make sure we understand the original rationale behind them first.

I found this as well, giving a very brief description https://superuser.com/questions/1193572/how-to-run-powerpoint-off-onedrive-in-browser-in-loop. The documentation link is broken though, so not too helpful at the moment 😷